crates/goose/src/providers/anthropic.rs

```diff
@@ -128,24 +123,17 @@ impl AnthropicProvider {
 #[async_trait]
 impl Provider for AnthropicProvider {
     fn metadata() -> ProviderMetadata {
+        let models: Vec<ModelInfo> = ANTHROPIC_KNOWN_MODELS
+            .iter()
+            .map(|&model_name| ModelInfo::new(model_name, 200_000))
+            .collect();
+
         ProviderMetadata::with_models(
             "anthropic",
             "Anthropic",
             "Claude and other models from Anthropic",
             ANTHROPIC_DEFAULT_MODEL,
-            vec![
-                ModelInfo::new("claude-sonnet-4-0", 200000),
```

Comment:
where to these live now - not used as it is always loaded?

Reply:
previously we declared ANTHROPIC_KNOWN_MODELS but were not using that at all. I now drag it in and just annotate it with 200k.

but yeah, the true way is for all providers to declare their models and token limits and then in models.rs read that to estimate limits instead of doing it twice or three times as we do now. small steps